src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputFormattedRightCurlySwitchCasesBlocks.java

```diff
@@ -77,17 +78,112 @@ public static void test3() {
     switch (k) {
       case 1:
         {
-          // violation above ''{' at column 9 should be on the previous line.'
           int x = 1;
           break;
         }
       case 2:
         {
-          // violation above ''{' at column 9 should be on the previous line.'
           int x = 2;
         }
         break;
       default:
+        {
+          int x = 2;
+        } // false-negative
```

Comment:
Why is it false negative?

Reply:
This is formatted file so formatter corrected `{}` placement, the comment was actually for a false negative case in the original file

https://github.com/checkstyle/checkstyle/blob/4d445eaf9bd774cd0d63e1febb3fa617c8281b09/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurlySwitchCasesBlocks.java#L89-L91

Reply:
I have removed such comments from the formatted file.